README.md

```diff
@@ -6,8 +6,6 @@ To use this container, you'll set environment variables in your docker run comma
 docker-collector-metrics uses those environment variables to generate a valid Metricbeat configuration for the container.
```

Comment:
`docker-collector-metrics is a Docker container that` --> `docker-collector-metrics is a Docker **image** that`
@imnotashrimp